examples/FileField/main.ipynb

```diff
@@ -59,7 +68,9 @@
    "id": "5277719b",
    "metadata": {},
    "source": [
-    "Then we define a simple web form to get the File we want to upload"
+    "##### 3. Define the webform\n",
+    "We define a simple webform for which will enable us to upload the file.\n",
+    "In the form definition below are two important things: **TextField** and **FileField**. These enable us to add custom text content and upload the file. You will see these components in the webform once deployed. You can add any other custom fields on your own."
```

Comment:
It would be nice if we had a reference doc with the list of custom fields that we support somewhere and to link to that doc.